eitprocessing/eit_data/loading/draeger.py

```diff
@@ -0,0 +1,266 @@
+from __future__ import annotations
+
+import sys
+import warnings
+from typing import TYPE_CHECKING, NamedTuple
+
+import numpy as np
+
+from eitprocessing.binreader.reader import Reader
+from eitprocessing.continuous_data import ContinuousData
+from eitprocessing.data_collection import DataCollection
+from eitprocessing.eit_data import EITData
+from eitprocessing.eit_data.event import Event
+from eitprocessing.eit_data.phases import MaxValue, MinValue
+from eitprocessing.eit_data.vendor import Vendor
+from eitprocessing.sparse_data import SparseData
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from numpy.typing import NDArray
+
+_FRAME_SIZE_BYTES = 4358
+FRAMERATE = 20
```

Comment:
To avoid confusion between modules, how about 

```suggestion
DRAEGER_FRAMERATE = 20
```